packages/tenant-process/src/services/tenantService.ts

```diff
@@ -669,6 +652,43 @@ async function assignCertifiedAttribute({
   return updatedTenant;
 }
 
+function buildVerifiedBy(
+  verifiedTenantAttribute: VerifiedTenantAttribute,
+  organizationId: TenantId,
+  tenantAttributeSeed: ApiVerifiedTenantAttributeSeed
```

Comment:
```suggestion
  verifiers: TenantVerifier[],
  organizationId: TenantId,
  expirationDate: string | undefined
```
You don't need to pass the whole attribute and seed, you only need these